src/components/Modals/ExecuteModal.js

```diff
@@ -149,42 +150,57 @@ export const ExecuteModal = ({
       isOpen={isOpen}
       onClose={onClose}
       isFooterLeftAligned
-      actions={[
-        <Button
-          key="confirm"
-          variant="primary"
-          ouiaId="etag"
-          isDisabled={connected.length === 0}
-          onClick={() => {
-            runRemediation(
-              remediationId,
-              etag,
-              disconnected.map((e) => e.executor_id).filter((e) => e)
-            );
-          }}
-        >
-          {isLoading
-            ? 'Execute playbook'
-            : `Execute playbook on ${pluralize(connectedCount, 'system')}`}
-        </Button>,
-        <Button
-          key="download"
-          variant="secondary"
-          ouiaId="download-playbook"
-          onClick={() => downloadPlaybook(remediationId)}
-        >
-          Download playbook
-        </Button>,
-        isDebug() ? (
-          <Button
-            key="reset-etag"
-            onClick={() => setEtag('test')}
-            ouiaId="reset-etag"
-          >
-            Reset etag
-          </Button>
-        ) : null,
-      ]}
+      actions={
+        systemCount !== 0
+          ? [
+              <Button
+                key="confirm"
+                variant="primary"
+                ouiaId="etag"
+                isDisabled={connected.length === 0}
+                onClick={() => {
+                  runRemediation(
+                    remediationId,
+                    etag,
+                    disconnected.map((e) => e.executor_id).filter((e) => e)
+                  );
+                }}
+              >
+                {isLoading
+                  ? 'Execute playbook'
+                  : `Execute playbook on ${pluralize(
+                      connectedCount,
+                      'system'
+                    )}`}
+              </Button>,
+              <Button
+                key="download"
+                variant="secondary"
+                ouiaId="download-playbook"
+                onClick={() => downloadPlaybook(remediationId)}
+              >
+                Download playbook
+              </Button>,
+              isDebug() ? (
+                <Button
+                  key="reset-etag"
+                  onClick={() => setEtag('test')}
+                  ouiaId="reset-etag"
+                >
+                  Reset etag
```

Comment:
typo in `etag`?

Reply:
not sure what this is, I don't think I wrote that directly

Reply:
etag is correct